_conferences/devfest-mn-2018.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "GDG DevFest MN"
```

Comment:
Please remove "MN". It is redundant with the location information.

Reply:
Will do – just don't tell the brand police, or they'll have my head

Reply:
Actually, is it okay if the name reflects the branding?
I got politely corrected about this already once this year for brand purposes. That's why I went with "DevFestMN" originally.

I'm totally fine following the ASG naming convention if that's a sticking point.

Reply:
I don't see any reason to deviate from the convention we already established. We haven't changed the even name from "Droidcon" despite their marketing being all lowercase and containing the location (e.g., "droidconNYC"). The site is a directory of events and since this is the same event type but in a different city I think uniformity wins.

Reply:
Sounds good, Jake.
I'm not trying to be difficult.
Let's do things the easy way, and folks that worry about brand can worry about it on their time.

If anything, I bet that your approach will be more helpful for ASG members, which matters a lot to me.

Reply:
Yep, I just wanted to clarify the thinking behind it.